### PR TITLE
Net6 Migration

### PR DESCRIPTION
**Pull request by ghost**
### Upgrades
* Migrated to Net 6
* Upgrade Grpc.AspNetCore.Web, protobuf-net.Grpc.AspNetCore, Grpc.Net.Client, Grpc.Net.Client.Web,  protobuf-net.Grpc packages to last stable version.

### New feature
* Fetch data page via grpc.

### CODE REVIEW

File: BlazorGrpcWebCodeFirst/Client/Pages/FetchData.razor [hunk 1]
```diff
@@ -36,12 +38,20 @@ else
...
     {
-        forecasts = await Http.GetFromJsonAsync<WeatherForecast[]>("WeatherForecast");
+        await LoadRemoteData();
+    }
+
+    async Task LoadRemoteData()
+    {
+        var reply = await WeatherForecastsService.Get();
+        forecasts = reply.Data;
     }
```
Comment (ghost) [inline BlazorGrpcWebCodeFirst/Client/Pages/FetchData.razor:57]: New feature, fetch data via gRPC protobuf

File: BlazorGrpcWebCodeFirst/Server/Services/WheatherService.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+using ProtoBuf.Grpc;
+using System.Threading.Tasks;
+using BlazorGrpcWebCodeFirst.Shared;
+using System;
+using System.Linq;
+
+namespace BlazorGrpcWebCodeFirst.Server.Services;
+
+#region snippet
+public class WheatherService : IWeatherForecasts
```
Comment (hakenr) [inline BlazorGrpcWebCodeFirst/Server/Services/WheatherService.cs]: Typo in service name, should be `WeatherService` without `h`.

**Review by hakenr — CHANGES_REQUESTED** [review]
Thank you for a great contribution!
...just one typo and we can merge the PR.

File: BlazorGrpcWebCodeFirst/Server/Services/WheatherService.cs (hunk 2 above)
Comment (ghost) [reply BlazorGrpcWebCodeFirst/Server/Services/WheatherService.cs]: Oh! thanks about review.

**Review by hakenr — APPROVED** [review]
LGTM

**Pull request merged**